docs/installation.rst

```diff
@@ -65,19 +65,50 @@ PyPi
 Executable
 ----------
 
-1. Download the executable from the GitHub release page: https://github.com/ScilifelabDataCentre/dds_cli/releases/latest/download/dds_cli_macos_x86_64
-   If you have an M1 Mac, you need to download this one instead: https://github.com/ScilifelabDataCentre/dds_cli/releases/latest/download/dds_cli_macos_arm64
-2. Open the terminal and go to the directory where the downloaded file is located
+1. Download the executable from the GitHub release page:
+
+   * for Linux: https://github.com/ScilifelabDataCentre/dds_cli/releases/latest/download/dds_cli_ubuntu_x86_64
+   * for MacOS: https://github.com/ScilifelabDataCentre/dds_cli/releases/latest/download/dds_cli_macos_x86_64
+   
+2. Open the terminal, go to the directory where the downloaded file is located, and make the file executable by running the following command:
+
+   .. code-block:: bash
+
+      $ chmod +x dds-cli_ubuntu_x86_64   
+
+   or
+
+   .. code-block:: bash
+
+      $ chmod +x dds-cli_macos_x86_64   
+
```

Comment:
Looks good but I think we should probably clarify that _if_ the file is not executable and they don't have permission, they need to do this.

Reply:
My impression is that the file is never executable, and also, if we write smth like _if the file is not executable_ then we need to add instructions about how to check that. Isn't it much simpler to just give them the _chmod_ command.

Reply:
Yeah true, just not sure we should encourage people to change permissions etc, but maybe I'm overly paranoid.